devicetypes/smartthings/zigbee-ceiling-fan-light.src/led-fan-lightings.groovy

```diff
@@ -20,15 +20,10 @@ import physicalgraph.zigbee.zcl.DataType
 import groovy.json.JsonOutput
 
 metadata {
-	definition (name: "LED FAN lightings", namespace: "SAMSUNG LED", author: "SAMSUNG LED", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true, genericHandler: "Zigbee") {
```

Comment:
You forgot about `genericHandler: "Zigbee"`

Reply:
Thank you so much :) I fixed it follow your advice.